recipes/sentry-native/all/conanfile.py

```diff
@@ -36,11 +36,9 @@ class SentryNativeConan(ConanFile):
         "qt": False,
         "with_crashpad": "sentry",
         "with_breakpad": "sentry",
-        "performance": False,
```

Comment:
Why not implement, instead of removing?

Reply:
Ok done here: [6b2d284](https://github.com/conan-io/conan-center-index/pull/10439/commits/6b2d284115eba93c4cd8aaa069ebed9e3c3ac472)